src/components/shipments/ShipmentReview.tsx

```diff
@@ -61,48 +52,38 @@ function ShipmentReview({
     return <UOLoader />;
   }
 
+  const additionalDetails: TableRowData[] = [
+    { label: 'Title', value: state.shipment.title },
+    { label: 'Status', value: state.shipment.status },
+    {
+      label: 'Proposal',
+      value: (
+        <Link
+          href="#"
+          onClick={() => history.push(`/ProposalEdit/${proposalData.id}`)}
```

Comment:
I think we could use the `react-router` `Link` component  so we can open the link like any other (e.g. new tab) and not relying on the `onClick`